### PR TITLE
Add matches screen; add random team name when create new team.

### PR DESCRIPTION
**Pull request by qingxin-yuan**

### CODE REVIEW

File: js/redux/modules/teams.js [hunk 1]
```diff
@@ -178,13 +179,20 @@ export const createTeamAndAddUser = (
 ) => async dispatch => {
   try {
     const uid = await AsyncStorage.getItem("user");
+<<<<<<< HEAD
     firebaseDB
+=======
+    const name = teamName(type);
+    let teamId = "";
+    await firebaseDB
+>>>>>>> Add matches screen; add random team name when create new team.
```
Comment (siddharthparmar7) [inline js/redux/modules/teams.js]: I guess you tried to remove some conflicts. Just make sure this is the code you want and remove the comments.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please check your merged code and inform me so I can approve this pull request. Thank you!

File: js/redux/modules/teams.js (hunk 1 above)
Comment (qingxin-yuan) [reply js/redux/modules/teams.js]: Changes done! Please review :)

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**